[ADD] Employees: New working hours doc (#19578)
odoo/documentation: content/applications/hr/employees/working_hours.rst

be selected per day. To add a time block, click in the desired day column at the hour the work time
starts. A time slot pop-up window displays with the following fields:

- :guilabel:`Duration`: Specify the total number of hours of the configured time slot.
- :guilabel:`Time`: Specify the start and end time of that time slot. Times are displayed in UTC.
- :guilabel:`Recurrency`: Enable to repeat the time slot to run every week. When disabled, the time
  slot does not repeat.
- :guilabel:`Repeat every`: Configure the number of days or weeks between occurrences, and
  optionally set an end date for the repetition. This field only displays when
  :guilabel:`Recurrency` is enabled.

The start and end times can be adjusted manually in the time slot window, and the
:guilabel:`Duration` field recalculates automatically based on these changes.

Click :guilabel:`Save` to confirm the time slot and add it to the calendar. After all the time slots
are configured for the variable working hours, click :icon:`fa-cloud-upload` :guilabel:`(Save
manually)`.

.. image:: working_hours/variable-schedule-type.png
   :alt: A variable schedule with four 4-hour blocks per day.

.. seealso::

>    **Employees**:

Reviewer: cool and innovative way to style seealso! :D

.. meta::
   :description: Working hours define the schedule during which employees or equipment are expected
                 to work or be available for use, and are configured separately for each company in
                 a database.

=============
Working hours
=============

In Odoo, *working hours* are the hours during which employees or equipment are scheduled to work or
be available for use.

.. note::
   Working hours may also be known as a *working schedule* in some Odoo applications.

Working hours are company-specific; however, working hours can be applied to all companies by
leaving the :guilabel:`Company` field blank.

Working hours appear in several apps, including:

- :menuselection:`Employees`
- :menuselection:`Payroll`
- :menuselection:`Planning`
- :menuselection:`Manufacturing`

> - :menuselection:`eCommerce`

Reviewer 2: ```suggestion
- :menuselection:`Employees app --> Configuration --> Working Schedules`
- :menuselection:`Payroll app --> Configuration --> Working Schedules`
- :menuselection:`Planning app --> Configuration --> Shift Templates`
- :menuselection:`Manufacturing app --> Configuration --> Work Centers`
- :menuselection:`eCommerce app --> Configuration --> Working Schedules`
```

Reviewer 2: IMO it would be helpful to clarify where working hours are found in each app!

Reviewer 2: also i'm guessing that
1. **planning** is the _shift templates_, and 
2. **manufacturing** is referring to the working hours form field selected on individual work centers?

if that's the case, it might be worth calling out that they are all shared in common with the exception of **planning** app's shift schedules (which i'm unsure about)

also also, 
3. i can't find it in ecommerce, am i missing something?

Working hours form
==================

> The following fields are available in all working hours forms:

Reviewer 2: for consistency, consider either
1. instructing the user to fill out the form fields, and using imperative at the start of each entry, or
2. keeping this line as-is and removing imperative language

Author: Some of it is imperative; some of it is auto-calculated by the other fields in the form being populated.

- :guilabel:`Name`: Enter the name for the new working schedule. This should be descriptive and
  clear to understand, such as `20 Hours/Week`.
- :guilabel:`Calendar Type`: Select :guilabel:`Fixed` or :guilabel:`Variable`. :ref:`Fixed working
  hours <employees/working_hours/fixed-schedule>` are good for schedules that require specific
  working hours. :ref:`Variable working hours <employees/working_hours/variable-schedule>` work best
  for businesses with split work schedules.

> - :guilabel:`Days per Week`: The number of days per week that the schedule is valid are displayed.

Reviewer 2: ```suggestion
- :guilabel:`Days per Week`: The number of days per week that the schedule is valid.
```

Reviewer 2: consider matching this to `hours per week`

- :guilabel:`Company`: Select the company that can use these new working hours from the drop-down

>   menu. A blank field indicates that it is available for all companies.

Reviewer 2: ```suggestion
  menu. Leaving this field blank indicates that it is available for all companies.
```

Reviewer 2: alt. empty

- :guilabel:`Hours per Week`: Specify the number of hours per week that the schedule covers.

.. _employees/working_hours/fixed-schedule:

> Fixed working hours

Reviewer 2: consider mentioning that fixed option is configured using a line item list/table -- it's not explicitly mentioned in text

-------------------

For the :guilabel:`Fixed` option, the :guilabel:`Hours per Week` and :guilabel:`Days per Week`
fields are automatic formula fields that update based on the number of hours configured in the
:guilabel:`Hours` column and the number of days listed in the :guilabel:`Day of the Week` column.

> When the work week days and hours are configured, click Save.

Reviewer 2: format `save` as guilabel!

The following columns are available in the form:

- :guilabel:`Day of Week`: Select the day of the week that the schedule is made for.
- :guilabel:`Hours`: Specify how many hours the schedule is for on the specified day.
- :guilabel:`Work from`: Specify the time that the schedule starts, in 24-hour format.
- :guilabel:`Work to`: Specify the time that the schedule ends, in 24-hour format.

If a column is not displayed, click the :icon:`fa-sliders` :guilabel:`(Optional column toggle)` icon

> and select the checkbox for it.

Reviewer 2: consider introducing the latter two bullets separately if they're not displayed by default, e.g.
> to additionally display...

.. important::
   Keep in mind, working hours are company-specific and **cannot** be shared between companies. Each

>    company needs to have its own set of working hours.

Reviewer 2: ```suggestion
Working hours are company-specific and **cannot** be shared between companies.
Each company needs to have its own set of working hours.
```

To edit a day and its hours, click in the :guilabel:`Hours` column and enter the number of working
hours for that day. To remove the day, click the :guilabel:`Delete row` icon. At the bottom of the
:guilabel:`Hours` column is the calculated total of work hours for the work week.

.. image:: working_hours/fixed-schedule-type.png
   :alt: A 40-hour, 5-day schedule of the Fixed calendar type.

.. _employees/working_hours/variable-schedule:

> Variable working hours

Reviewer 2: consider **leading** with fact that variable option is configured using the calendar view

Author: I feel like that's implied with "variable calendar types."

----------------------

Variable calendar types are well-suited to businesses with split work schedules. This option allows
industries with variable demand, such as hotels or businesses that rely heavily on tourism, to staff
peak hours while reducing labor costs during slow midday periods.

When the :guilabel:`Variable` option is chosen, a calendar view displays. The calendar allows the

> user to select multiple time blocks per day for work time. To add a time block, click in the desired

Reviewer 2: ```suggestion
When the :guilabel:`Variable` option is chosen, a calendar view displays. Multiple time blocks can be selected per day. To add a time block, click in the desired
```

Reviewer 2: to address reader directly instead of mentioning "user"

adjust if not using imperative

day column at the hour the work time starts. A time slot pop-up window displays with the following

> fields:

Reviewer 2: similar to above, either instruct here and use imperatives in each entry, or remove imperatives from the entries

> - :guilabel:`Duration`: Represents the total number of hours of the configured time slot.

Reviewer 2: ```suggestion
- :guilabel:`Duration`: The total number of hours of the configured time slot.
```

> - :guilabel:`Time`: Represents the start and end time of that time slot. It uses UTC.

Reviewer 2: ```suggestion
- :guilabel:`Time`: The start and end time of that time slot. It uses UTC.
```

> - :guilabel:`Recurrency`: Enables the time slot to run every week. When disabled, the time slot does

Reviewer 2: ```suggestion
- :guilabel:`Recurrency`: Enable to repeat the time slot to run every week. When disabled, the time slot does
```

Reviewer 2: adjust based on decision to use/not use imperatives

  not repeat.
- :guilabel:`Repeat every`: When :guilabel:`Recurrency` is enabled, this field displays. Users can
  configure the number of days or weeks between occurrences and whether there is an end date for the

>   repetition.

Reviewer 2: ```suggestion
- :guilabel:`Repeat every`: Configure the number of days or weeks between occurrences, and optionally set an end date for the repetition. This field only displays when :guilabel:`Recurrency` is enabled.
```

Reviewer 2: flip to provide useful details first, and add conditional after. preferably avoid "user" language, adjust if not using imperative

The start and end times can be adjusted manually in the time slot window, and the
:guilabel:`Duration` field recalculates automatically based on these changes.

Enable the :guilabel:`Recurrency` option if the time slot recurs. To repeat the time slot:

#. Enter a number and select :guilabel:`Days` or :guilabel:`Weeks` from the drop-down menu.
#. Choose how the repetition ends. Select :guilabel:`Forever` if there is no end date. Select
   :guilabel:`Occurrences` to enter how many times it should repeat. Select :guilabel:`Until` to

>    choose an end date from a calendar pop-up window.

Reviewer 2: this is a bit redundant with the field descriptions above, consider
1. deleting this and just keeping info to the field entries, or
2. simplifying the entries, and linking to a sub-heading here

Click :icon:`fa-cloud-upload` :guilabel:`(Save manually)` to confirm the time slot and add it to the

> calendar. After all the time slots are configured for the calendar, click :icon:`fa-cloud-upload`

Reviewer 2: ```suggestion
calendar. After all the time slots are configured for the variable working hours, click :icon:`fa-cloud-upload`
```

:guilabel:`(Save manually)`.

.. image:: working_hours/variable-schedule-type.png

>    :alt: A variable schedule with four 4-hour blocks per day.

Reviewer 2: the hours per week says 10h40m/day? four 4-hour blocks would be 16h/day

.. seealso::
   - :doc:`new_employee`
   - :doc:`../payroll/contracts`
   - :doc:`../payroll/working_schedules`
   - :doc:`../payroll/salaries`
   - :doc:`../../services/planning`
   - :doc:`../../inventory_and_mrp/manufacturing/advanced_configuration/using_work_centers`
   - :doc:`../../inventory_and_mrp/manufacturing/workflows/work_center_time_off`

>    - :doc:`eCommerce: Delivery methods <../../websites/ecommerce/shipping>`

Reviewer 2: consider splitting this by app, or at least identifying the app so it's less ambiguous
